docs/developer-docs/latest/plugins/graphql.md

```diff
@@ -385,10 +385,19 @@ When [extending the GraphQL schema](#extending-the-schema), the `resolversConfig
 
 ##### Authorization configuration
 
-<!-- TODO: consider reworking the TOC as having h5s is not optimal… -->
-By default, the authorization of a GraphQL request is handled through the [Users & Permissions plugin](#usage-with-the-users-permissions-plugin). A request is allowed if the appropriate permissions are given: for instance, if a 'Category' content-type exists and is queried through GraphQL with the `Query.categories` handler, the request is allowed if the appropriate `find` and/or `findOne` permission(s) for the 'Categories' content-type are given.
+By default, the authorization of a GraphQL request is handled by the registered authorization strategy that can be either [API token](/developer-docs/latest/setup-deployment-guides/configurations/required/admin-panel.md#api-tokens) or through the [Users & Permissions plugin](#usage-with-the-users-permissions-plugin). The Users & Permissions plugin offers a more granular control.
 
-To change how this authorization is configured, use the `resolversConfig.auth` key that accepts:
+::: details Authorization with the Users & Permissions plugin
+With the Users & Permissions plugin, a GraphQL request is allowed if the appropriate permissions are given.
+
+For instance, if a 'Category' content-type exists and is queried through GraphQL with the `Query.categories` handler, the request is allowed if the appropriate `find` permission for the 'Categories' content-type is given.
+
+To query a single category, which is done with the `Query.category` handler, the request is allowed if the the `findOne` permission is given.
+
+Please refer to the user guide on how to [define permissions with the Users & Permissions plugin](/user-docs/latest/users-roles-permissions/configuring-administrator-roles.md#editing-a-role).
+:::
+
+To change how the authorization is configured, use a `auth` attribute in the resolver configuration defined at `resolversConfig.[MyResolverName]`. The `auth` attribute accepts:
 
 * either `false` to fully bypass the authorization system and allow all requests,
 * or a `scope`, as an array of strings, to define the scopes (i.e. permissions) that the request needs to have to be allowed.
```

Comment:
The wording led me to think that `auth` can be a scope defined in an array, where in fact it's an object containing a `scope` attribute (which is an array). Do you think it would be possible to maybe make it clearer?